### DISCUSSION TITLE
MacOS Download/Upload UI

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#2991] [Q&A]
Is there a reason why the icon UI has the download value above the upload value when logic says up is up and down is down?

![Screenshot 2021-12-10 at 7 39 37 PM](https://user-images.githubusercontent.com/69029666/145529188-3cf6180e-5022-4ab3-80b2-a64b0bac19ad.png)

**Comment by Jfischthecat** [comment]
Download speed is more important to most, therefore it goes on top.
That's my guess.
